src/pybind/mgr/CMakeLists.txt

```
@@ -11,23 +18,44 @@ endif()
 
 # Location needs to match default setting for mgr_module_path, currently:
 # OPTION(mgr_module_path, OPT_STR, CEPH_INSTALL_DATADIR "/mgr")
-install(DIRECTORY
-  ${CMAKE_CURRENT_SOURCE_DIR}
-  DESTINATION ${CEPH_INSTALL_DATADIR}
-  REGEX "CMakeLists.txt" EXCLUDE
-  REGEX "\\.gitignore" EXCLUDE
-  REGEX ".*\\.pyi" EXCLUDE
-  REGEX "hello/.*" EXCLUDE
-  REGEX "tests/.*" EXCLUDE
-  REGEX "rook/rook-client-python.*" EXCLUDE
-  REGEX "osd_perf_query/.*" EXCLUDE
-  REGEX "tox.ini" EXCLUDE
-  REGEX "requirements.*\.txt" EXCLUDE
-  REGEX "constraints.*\.txt" EXCLUDE
-  REGEX "node_modules" EXCLUDE
-  REGEX "cypress.*" EXCLUDE
-  REGEX "\.coveragerc" EXCLUDE
-  REGEX "\.editorconfig" EXCLUDE
-  REGEX "\..*lintrc" EXCLUDE
-  REGEX "\.browserslistrc" EXCLUDE
-  REGEX "\.prettier*" EXCLUDE)
+set(mgr_modules
+  alerts
+  balancer
+  cephadm
+  crash
+  # dashboard (optional)
+  devicehealth
+  diskprediction_local
+  # hello is an example for developers, not for user
+  influx
+  insights
+  iostat
+  k8sevents
+  localpool
+  mds_autoscaler
+  mirroring
+  nfs
+  orchestrator
+  # osd_perf_query (for testing purpose only)
+  osd_support
+  pg_autoscaler
+  progress
+  prometheus
+  rbd_support
+  restful
+  # rook (optional)
+  selftest
+  snap_schedule
+  stats
+  telegraf
+  telegraf
+  # tests (for testing purpose only)
+  test_orchestrator
+  volumes
+  zabbix)
+
+install(DIRECTORY ${mgr_modules}
+  DESTINATION ${CEPH_INSTALL_DATADIR}/mgr
+  ${mgr_module_install_excludes})
```

Why not installing all by default except the specific dashboard/test/... dirs?
```suggestion
install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
DESTINATION ${CEPH_INSTALL_DATADIR}
REGEX "dashboard/.*" EXCLUDE
REGEX "osd_perf_query/.*" EXCLUDE
REGEX "rook/.*" EXCLUDE
REGEX "tests/.*" EXCLUDE
```

---

because i wanted to have an "include" list not exclude list. IMHO, the former is more explicit .